src/Algebra/Structures.agda

```diff
@@ -425,6 +425,15 @@ record IsCommutativeSemiring (+ * : Op₂ A) (0# 1# : A) : Set (a ⊔ ℓ) where
     }
 
 
+record IsCommutativeCancellativeSemiring (+ * : Op₂ A) (0# 1# : A) : Set (a ⊔ ℓ)
```

Comment:
Please follow the layout of the other record declarations in the file. The problems currently are:
 - the `where` not being on the same line
 - the fields not being indented underneath the `field` keyword